docs/challenges.rst

```diff
@@ -1,159 +1,86 @@
+  plugin if you have the technical knowledge to make configuration changes yourself when asked to do so. 
+  Note that this will prevent automated renewal of your certificate using `certbot renew`.
+
+Tips for Challenges
+-------------------
+General tips:
+
+* Run Certbot on your web server, not on your laptop or another server. It’s usually the easiest way to get a certificate.
+* Use a tool like the DNSchecker at dnsstuff.com to check your DNS records to make sure 
+  there are no serious errors. A DNS error can prevent a certificate authority from 
+  issuing a certificate, even if it does not prevent your site from loading in a browser.
+* If you are using Apache or NGINX plugins, make sure the configuration of your Apache or NGINX server is correct.
+
+HTTP-01 Challenge
+~~~~~~~~~~~~~~~~~
+
+* Make sure the domain name exists and is already pointed to the public IP address of the server where 
+  you’re requesting the certificate.
+* Make sure port 80 is open, publicly reachable from the Internet, and not blocked by a router or firewall.
+* When using the Webroot plugin or the manual plugin, make sure the the webroot directory exists and that you
+  specify it properly. If you set the webroot directory for example.com to `/var/www/example.com`
+  then a file placed in `/var/www/example.com/.well-known/acme-challenge/testfile` should appear on
+  your web site at `http://example.com/.well-known/acme-challenge/testfile` (A redirection to HTTPS
   is OK here and should not stop the challenge from working.)
+* In some web server configurations, all pages are dynamically generated by some kind of framework, 
+  usually using a database backend. In this case, there might not be a particular directory 
+  from which the web server can serve filesdirectly. Using the Webroot plugin in this case 
+  requires making a change to your web server configuration first.
+* Make sure your web server serves files properly from the directory where the challenge 
+  file is placed (e. g. `/.well-known/acme-challenge`) to the expected location on the 
+  website with the proper MIME type and without adding a header or footer.
+* When using the Standalone plugin, make sure another program is not already listening to port 80 on the server.
+* When using the Webroot plugin, make sure there is a web server listening on port 80.
+
+TLS-SNI-01 Challenge
+~~~~~~~~~~~~~~~~~~~~
+
+* The TLS-SNI-01 challenge doesn’t work with content delivery networks (CDNs) 
+  like CloudFlare and Akamai because the domain name is pointed at the CDN, not directly at your server.
+* Make sure port 443 is open, publicly reachable from the Internet, and not blocked by a router or firewall.
+* When using the Apache plugin, make sure you are running Apache and no other web server on port 443.
+* When using the NGINX plugin, make sure you are running NGINX and no other web server on port 443.
+* With either the Apache plugin or the NGINX plugin, check whether certbot is modifying your web 
```

Comment:
I'd re-word it in such a way as to not suggest users look at their configurations if certbot doesn't show an error. Instead I'd say something like:
"With either the Apache or NGINX plugin certbot will also automatically modify your web server configuration to serve the certificate. If you'd prefer this not to happen you can use the ``certonly`` command to just obtain the certificate, then install it manually. If you receive an InstallationError that means you have received a certificate but the plugin was unable to modify your web server configuration, meaning that you'll have to install it manually."
But maybe that's too wordy?
Essentially you should let people know that if it runs the challenge but gives an error on install then they still have a valid cert, but prompting people to read certbot's changes to their apache/nginx configs when things are fine seems like a bad idea.